paper_notes/object_detection_based_on_region_decomposition.md

```diff
@@ -0,0 +1,35 @@
+# [Object Detection based on Region Decomposition and Assembly](https://arxiv.org/pdf/1901.08225.pdf)
+
+_May 2019_
+
+tl;dr: Summary of the main idea.
+This paper propose a region decomposition and assembly(R-DAD) for more accurate object detection.
```

Comment:
Add motivation of the paper: to tackle occlusion.

Add this link to the `Notes` session
https://blog.csdn.net/qq_30708445/article/details/88182603